osu.Game/Localisation/DeleteConfirmationContentStrings.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Localisation;
+
+namespace osu.Game.Localisation
+{
+    public static class DeleteConfirmationContentStrings
+    {
+        private const string prefix = @"osu.Game.Resources.Localisation.DeleteConfirmationContent";
+
+        /// <summary>
+        /// "All beatmaps?"
+        /// </summary>
+        public static LocalisableString Beatmaps => new TranslatableString(getKey(@"beatmaps"), @"All beatmaps?");
```

Comment:
I think the message should be "Are you sure you want to delete all beatmaps", and the title of the notification should be "Caution". This will allow for better localisation support. Separating out just the "Delete" word won't work in many languages.